python_ta/checkers/function_parameter_not_mentioned_checker.py

```diff
@@ -0,0 +1,59 @@
+"""checker that every function parameter is mentioned by name in the docstring text.
+"""
+
+from __future__ import annotations
+
+from typing import Optional
+
+from astroid import nodes
+from pylint.checkers import BaseChecker
+from pylint.checkers.utils import only_required_for_messages
+from pylint.lint import PyLinter
+
+
+class FunctionParameterNotMentionedChecker(BaseChecker):
+    """
+    A class checker to check if every function parameter is mentioned by name within it's the docstring.
+    By default, this checker will be disabled.
+    """
+
+    name = "function_parameters_not_mentioned"
+    msgs = {
+        "C9960": (
+            "The parameter '%s' is not mentioned in the docstring",
+            "function-parameters-not-mentioned",
+            "Used when a function parameter is not mentioned in the docstring",
+        )
+    }
+
+    def __init__(self, linter: Optional[PyLinter] = None) -> None:
+        super().__init__(linter=linter)
+
+    @only_required_for_messages("function-parameters-not-mentioned")
+    def visit_functiondef(self, node: nodes.FunctionDef) -> None:
+        """Visit a function definition"""
+        docstring = node.doc_node.value if node.doc_node and node.doc_node.value else ""
+        self._check_parameters(self._strip_docstring_of_doctest(docstring), node.argnames(), node)
```

Comment:
You should provide a screenshot of what the error message looks like.

In particular, passing in `node` isn't ideal; this will cause the entire function definition to be highlighted, but really we should only highlight the specific parameter that isn't mentioned.

So instead of using `node.argnames()`, you'll need to figure out how to iterate over the function's parameters in the AST.